web/src/layouts/MainLayout/MainLayout.js

```diff
@@ -24,6 +24,7 @@ import logo from 'src/layouts/MainLayout/Logo_2.jpg'
 const MainLayout = ({ children }) => {
   const { logIn, logOut, isAuthenticated, currentUser } = useAuth()
   const { data, loading } = useQuery(QUERY, {
+    skip: !currentUser?.sub,
```

Comment:
Only found this in the docs today. Since react hooks like `useQuery` have to be at the top level of the function and can be put into normal if statements, this kinda logic has to be built into the hook itself. But yeah found what I needed pretty quickly looking at the apollo dogs.